### PR TITLE
FEAT: adding sticky notes as a keybind

### PR DESCRIPTION
**Pull request by DMelisena**
# Pull Request

## Description

A simple sticky notes feature. implementation of https://github.com/prasanthrangan/hyprdots/issues/1635 feature request. this will :

1. install https://github.com/vixalien/sticky from aur as a custom install and keybind it to Super+grave
2. keybind it as a script that check using pgrep, it will spawn if its not opened and close if its already opened
3. customize the apps as float on windows rule

ps: forgive me for not using the template on commit guidelines. I tried to comply with the contribution guideline but seems to miss it out.
this use the same key as quickphrase feature, i think from fcitx5 and will replace it. I never use the quickphrase feature myself though. I asked around and it seems that no one use it anyway.

## Type of change

Please put an `x` in the boxes that apply:

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [X ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [ ] **Documentation update** (non-breaking change; modified files are limited to the documentations)
- [ ] **Technical debt** (a code change that does not fix a bug or add a feature but makes something clearer for devs)
- [ ] **Other** (provide details below)

## Checklist

Please put an `x` in the boxes that apply:

- [x] I have read the [CONTRIBUTING](https://github.com/prasanthrangan/hyprdots/blob/main/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My commit message follows the [commit guidelines](https://github.com/prasanthrangan/hyprdots/blob/main/CONTRIBUTING.md#git-commit-messages).
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added a changelog entry.
- [ ] I have added necessary comments/documentation to my code.
- [ ] I have added tests to cover my changes.
- [ ] I have tested my code locally and it works as expected.
- [ ] All new and existing tests passed.

## Screenshots

240629_23h50m46s_screenshot

## Additional context

I missed out the commit message guideline, sorry. I also dont know where window rule should be implemented except on opacity. I Will suggest a documentation update if this seems to be applicable. This is my first try on contributing to an opensource so i wish for guidance.

### CODE REVIEW

**Comment by Enigma1309** [comment]
can we add support for flatpak app too?

File: Scripts/custom_hypr.lst [hunk 1]
```diff
@@ -59,3 +59,4 @@ pokemon-colorscripts-git|zsh
 lsd|fish
 starship|fish
 hyprdots-ctl
+aur:sticky-notes                                       # ide text editor
```
Comment (kRHYME7) [inline Scripts/custom_hypr.lst:62]: Should be in ` ./custom_apps.lst ` for optional packages

**Review by kRHYME7 — CHANGES_REQUESTED** [review]

**Comment by mahaveergurjar** [comment]
Please add this package in `./custom_apps.lst` don't add in `./custom_hypr.lst`

**Comment by T-Crypt** [comment]
Fix conflicts and add package to custom_apps.lst

**Pull request closed without merging**